File: app/controllers/ComparingHandler.java
package controllers;


import model.StringElement;

public class ComparingHandler implements Runnable{

	private final StringElement element;
	private final int position;

	public ComparingHandler(StringElement element, int i) {
		super();
		this.element = element; 
		position = i;
	}

	@Override
	public void run() {
		Main.checkElementIsInOtherStringsXML(element, position);
	}

}

File: app/controllers/ComparingHandler.scala
package controllers

import model.StringElement
/**
 * User: Kayrnt
 * Date: 08/10/13
 * Time: 16:49
 */

class ComparingHandler(elementC : StringElement,
                       i : Int,
                       process : StringToolProcess)
  extends Runnable{

var element : StringElement = elementC
var position : Int = i
var stringProcess : StringToolProcess = process

  def run() {
    stringProcess.checkElementIsInOtherStringsXML(element, position);
  }

}
